configs/mmcls/mmcls_regression_test.yaml

```diff
@@ -0,0 +1,78 @@
+globals:
+  codebase_name: mmcls
+  codebase_dir: ../mmclassification
+  checkpoint_force_download: False
+  checkpoint_dir: ../mmdeploy_checkpoints
+  images:
+    img_coco_320x320: &img_coco_320x320 ./tests/data/tiger.jpeg
```

Comment:
may only need two image of shape 224x224

Reply:
Done